src/mlpack/tests/customlayer.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef CUSTOMLAYER_HPP
+#define CUSTOMLAYER_HPP
+
+#include <mlpack/prereqs.hpp>
+#include <mlpack/methods/ann/layer/layer.hpp>
+
+
+namespace mlpack {
+namespace ann{
```

Comment:
I guess the style checker did not catch it, but if you can put a space between `ann` and `{` I'd appreciate it. :)

Reply:
No problem, will make the changes. Thanks for looking into it. I removed the include to specifically show that the visitors are not dependent on the LayerTypes. Hopefully it will also help reduce triggering compilations when not required. I need to run some local tests for the layer types that you mentioned. Will send an update when ready.